### ISSUE TITLE
[Future request] Allow user to limit number of cores the render can utilize

### ISSUE THREAD

**Issue by Svarov-RZM** [synfig/synfig#1410]
**Description**
The rendering engine hogs all available CPU cores by default. I think it would be good to allow user to limit the amount of cores Synfig uses. For example, I usually listen to various playlists on YouTube while I'm working and in some cases those CPU spikes to 100% when Synfig renders current frame is noticeable.

It would be good to have such setting in "Preferences->Render" dialog, something like "Limit render to number of cores".

**Comment by ice0** [comment]
I agree. As a temporary solution, you can limit the number of rendering threads using the environment variable `SYNFIG_GENERIC_THREADS`.

Just run "SYNFIG_GENERIC_THREADS = 2 ./synfigstudio" on Linux or set it in the settings on Windows.

**Comment by SushantAA** [comment]
I want to work on this issue 
pls assign me 
is there any chat Chanel

**Comment by ice0** [comment]
Hi and welcome here!
Assigned to you.

P.S.
Unfortunately there is no chat here.
Btw, you can ask here or at https://forums.synfig.org

**Comment by SushantAA** [comment]
thanks :smile:

**Comment by DhairyaBahl** [comment]
@ice0 PR related to the issue is merged ! Is it resolved ? If yes then kindly close it :) If not, then kindly assign it to me.

Thanks and Regards :)
